### PR TITLE
New icon

### PR DESCRIPTION
**Pull request by Ryzhkov-Alexey**
new icons

### CODE REVIEW

**Review by rzvdv — CHANGES_REQUESTED** [review]
There are too many changes in package-json.lock, could you delete that file from commit ?

File: src/styles/messages.scss [hunk 1]
```diff
@@ -166,7 +166,7 @@
 		&.json {
 			position: relative;
 			&:before {
-				content: '{~}';
+				content: url(../../resources/icons/json.svg);
```
Comment (rzvdv) [inline src/styles/messages.scss:169]: json icon aligning seems to be a litte off

Comment (Ryzhkov-Alexey) [reply src/styles/messages.scss:169]: I will fix it, thanks

**Review by rzvdv — CHANGES_REQUESTED** [review]
Same icons are used in RuleEditor component (settings icon on top right). We need to change them aswell and delete old icons

**Review by rzvdv — APPROVED** [review]

**Pull request merged**